spec/modules-spec.js

```diff
@@ -47,6 +47,20 @@ describe('third-party module', function () {
       })
     })
   })
+
+  describe('global variables', function () {
+    describe('process', function () {
+      it('can be declared in a module', function () {
+        require(path.join(fixtures, 'module', 'declare-process.js'))
```

Comment:
Sure thing. I borrowed the path.join call from elsewhere in this file - do you have a preference either way?